content/academy/glossary/proxyman.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Proxyman
+description: Learn about Proxyman, a tool for viewing all network requests that are coming through your system. Filter by response type, by a keyword, or by application.
+menuWeight: 6.3
+paths:
+    - glossary/proxyman
```

Comment:
I made this, but I don't like it. I don't like how all the glossary terms are nested under the "Web scraping glossary" top level menu.

Maybe we could keep the "GLOSSARY" as category, but change the dropdowns to "Tools" and "Concepts" and then put Proxyman and Postman under Tools and Rate-limiting under Concepts?